dbus/methods.go

```diff
@@ -196,7 +196,21 @@ func (c *Conn) KillUnit(name string, signal int32) {
 
 // KillUnitContext same as KillUnit with context
 func (c *Conn) KillUnitContext(ctx context.Context, name string, signal int32) {
-	c.sysobj.CallWithContext(ctx, "org.freedesktop.systemd1.Manager.KillUnit", 0, name, "all", signal).Store()
+	c.KillSpecificProcessUnitContext(ctx, name, All, signal)
+}
+
+type Who string
```

Comment:
This (and the constants below) is a public entry in the API, and thus would need docstrings. Also, it should float to the top of the file.